api/src/nv_ingest_api/internal/schemas/extract/extract_pdf_schema.py

```diff
@@ -47,7 +47,7 @@ class PDFiumConfigSchema(BaseModel):
     yolox_infer_protocol: str = ""
 
     nim_batch_size: int = 4
-    workers_per_progress_engine: int = 5
+    workers_per_progress_engine: int = 16
```

Comment:
@drobison00 ,  Is this the best way to modify this , with the improved page elements v2 nim, bottleneck of this stage moves to pdf extraction and i want to use workers_per_progress_engine processes per pdf to read them into images.

Reply:
On this PR, this is what i see for really long PDF's 

<img width="1204" alt="image" src="https://github.com/user-attachments/assets/005f745b-49ff-44fa-8b0b-7beda788a11f" />

<img width="1204" alt="image" src="https://github.com/user-attachments/assets/754e2ad2-cff3-4228-a305-8dc77c487114" />